controllers/database_controller.py

```diff
@@ -61,3 +62,40 @@ def get_iac_for_collection_exercise(collection_exercise_id, social=False):
     for row in result:
         iac = row['iac']
     return iac
+
+
+def get_all_iacs_for_collection_exercise(collection_exercise_id, social=False):
```

Comment:
is `get_iac_for_collection_exercise` now dead unused code?

Reply:
Nope used in multiple places